### ISSUE TITLE
How can I always use the latest git version of mpv?

### ISSUE THREAD

**Issue by Jimmy03** [smplayer-dev/smplayer#152]
The git binaries for mpv are here:  
https://sourceforge.net/projects/mpv-player-windows/files/64bit/

Can I just put these into the smplayer mpv folder?

I'm just using the standard releases of smplayer, not the dev

I've been on 20.6 for a long time

**Comment by smplayer-dev** [comment]
Yes, you can replace mpv with a new one. But if you're using an old version of smplayer then I can't tell if it would work.
